tests/test-email-prepare.php

```diff
@@ -0,0 +1,94 @@
+<?php
+include_once dirname( __FILE__ ) . '/email-test-case.php';
+
+/**
+ * Class testing Caldera_Forms_Email_Prepare
+ */
+class Test_Email_Prepared extends Email_Test_Case {
+
+
+	/**
+	 * test return email from rfc822
```

Comment:
@robwilde This inline documentation is insufficient.

Sentences start with capital letters, all methods need @since tags and all tests need @covers tags

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/

https://phpunit.de/manual/current/en/appendixes.annotations.html#appendixes.annotations.covers

Reply:
https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/ more specifically

Reply:
updating this now